apps/scan/backend/scripts/copy_sheets.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import path from 'path';
+import { v4 as uuid } from 'uuid';
+import { AcceptedSheet } from '@votingworks/backend';
+import {
+  assert,
+  assertDefined,
+  extractErrorMessage,
+} from '@votingworks/basics';
+import { safeParseNumber } from '@votingworks/types';
+
+import { SCAN_WORKSPACE } from '../src/globals';
+import { Store } from '../src/store';
+
+const usageMessage = `Usage: copy-sheets <target-sheet-count>
+
+Arguments:
+  <target-sheet-count>\tThe target total sheet count after copying`;
+
+interface CopySheetsInput {
+  targetSheetCount: number;
+}
+
+function checkEnvironment(): void {
+  assert(process.env.NODE_ENV !== 'production', 'Cannot be run in production');
+}
+
+function parseCommandLineArgs(args: readonly string[]): CopySheetsInput {
+  if (args.length !== 1 || !safeParseNumber(args[0]).isOk()) {
+    console.log(usageMessage);
+    process.exit(0);
+  }
+  const targetSheetCount = safeParseNumber(args[0]).unsafeUnwrap();
+  return { targetSheetCount };
+}
+
+function copySheet(store: Store, sheet: AcceptedSheet): void {
+  const newSheetId = uuid();
+  const newSheet: AcceptedSheet = {
+    ...sheet,
+    id: newSheetId,
+    frontImagePath: sheet.frontImagePath.replace(sheet.id, newSheetId),
+    backImagePath: sheet.backImagePath.replace(sheet.id, newSheetId),
```

Comment:
I'm not sure we want to assume that the image path has the sheet ID in it here, even though that is the case right now. Consider either calling the same logic that generates the path in the first place _or_ just generates a unique name in the same directory?

Reply:
Good call: https://github.com/votingworks/vxsuite/pull/4225/commits/c083493de8335a3e153306abbc43fa484aded67f